### PR TITLE
[ENH]: GUI - Synchronous Inputs

### PR DESCRIPTION
**Pull request by kmilo9999**
1. Added gui support for Synchronous Inputs in drivers simulations
2. Checkbox initial value loaded from values in default.json
Solves #716
image
image

### CODE REVIEW

**Comment by kmilo9999** [comment]
@ntolley We were looking into loading the init synchronous input values from the json file, and we found that the function `_extract_drive_specs_from_hnn_params` (hnn_core/params.py) hardcodes the value for `cell_specific` to False. Given that, should we always initialize the synch inputs widget unchecked ?

**Review by gtdang — APPROVED** [review]
This looks good to me!

File: hnn_core/gui/gui.py [hunk 1]
```diff
@@ -854,6 +863,11 @@ def _get_poisson_widget(name, tstop_widget, layout, style, location, data=None,
                        layout=layout,
                        style=style)
 
+    synch_inputs = Checkbox(value=sync_evinput,
```
Comment (jasmainak) [inline hnn_core/gui/gui.py:866]:
```suggestion
    sync_inputs = Checkbox(value=sync_evinput,
```

just to be a little more consistent :)

**Comment by jasmainak** [comment]
Looks good other than minor nitpick, don't forget to update `whats_new.rst` !

**Comment by jasmainak** [comment]
Thanks @kmilo9999 !! Great work 🥳 🍺

**Pull request merged**